.github/workflows/jira.yaml

```diff
@@ -77,11 +62,11 @@ jobs:
       uses: atlassian/gajira-transition@v2.0.1
       with:
         issue: ${{ steps.search.outputs.issue }}
-        transition: Done
+        transition: Close
```

Comment:
Since "Close"'ing is immediately followed by a prompt that requires you to choose how it's been addressed, does closing programmatically this way work?

Reply:
I don't believe so.  Did it in the past with `Done`?

Reply:
Never mind, re-read your comment. I am going to check a PR that I recently resolved in TFVP.

Reply:
@calvn `VAULT-4202` went  directly to `CLOSED`. No resolution provided.